main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Arguments.java

```diff
@@ -0,0 +1,18 @@
+package org.mobilitydata.gtfsvalidator.cli;
+
+import com.beust.jcommander.Parameter;
+
+/**
+ * Command-line arguments for GTFS Validator CLI.
+ */
+public class Arguments {
+
+    @Parameter(names = {"-i", "--input"}, description = "Location of the unarchived directory with input GTFS files", required = true)
```

Comment:
I provided the file name `gtfs.zip` as the `-i` parameter and that worked. Can you provide a base directory that contains multiple zip files too? Suggest clarifying the description of this parameter.

Reply:
This was an outdated comment. Fixed.